packages/synapse-react-client/src/components/download_list/TableQueryDownloadConfirmation.test.tsx

```diff
@@ -44,14 +44,24 @@ const mockUseGetQueryResultBundle = jest.mocked(
   useGetQueryResultBundleWithAsyncStatus,
 )
 const mockUseAddQueryToDownloadList = jest.mocked(useAddQueryToDownloadList)
-
+const ID_COLUMN_ID = 11112
```

Comment:
Set up a couple of ColumnModels with the response data - one column for 'id' and one for 'currentVersion'.  Then verify that when fileIdColumnName and fileVersionColumnName are set, the selectFileColumn and selectFileVersionColumn are set in the AddToDownloadListRequest